frontend/src/components/navigation/MobileMenu.tsx

```
@@ -0,0 +1,217 @@
+              aria-label="Close menu"
+              onClick={onClose}
+              className="p-2 text-gray-500 hover:text-gray-700 hover:bg-gray-100 rounded-md transition-colors focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-600"
+            >
+              <span aria-hidden="true" className="text-2xl leading-none">
+                ×
+              </span>
+            </button>
+          </div>
+
+          {/* Navigation */}
+          <nav role="navigation" aria-label="Main navigation" className="py-4">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<nav>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)